config.yaml.defaults

```diff
@@ -46,7 +46,8 @@ app:
       component: RunSummary
     - path: "/share_data/:id"
       component: ShareDataForm
-
+    - path: "/group_sources/:id"
```

Comment:
Can we add a component for this page to render in the sidebar?

Reply:
I don't know what that means...

Reply:
I thought the natural thing is to add a link from the group page...

Reply:
I agree with @guynir42